ddtrace/internal/packages.py

```diff
@@ -77,10 +77,11 @@ def get_module_distribution_versions(module_name: str) -> t.Optional[t.Tuple[str
     pkgs = get_package_distributions()
     while names == []:
         try:
-            return (
-                module_name,
-                importlib_metadata.distribution(module_name).version,
-            )
+            package = importlib_metadata.distribution(module_name)
+            name = package.metadata["name"]
+            version = package.version
```

Comment:
```suggestion
            metadata = package.metadata
            name = metadata["name"]
            version = metadata["version"]
```

Every time you access a property on a distribution it'll re-parse the metadata file 🤦🏻

Reply:
Oh good to know !